.travis.yml

```diff
@@ -124,19 +124,6 @@ matrix:
       before_script:
         - psql -c 'create database sentry;' -U postgres
 
-    # only the sqlite suite runs riak tests
-    - python: 2.7
-      name: 'Backend [SQLite]'
-      env: TEST_SUITE=sqlite DB=sqlite
-      services:
-        - memcached
-        - riak
```

Comment:
@mattrobenolt Do we need something that tests Riak still?

Reply:
Ideally for now since we still use it in production. But it also never changes, so.